tests/scripts/thread-cert/Cert_7_1_03_BorderRouterAsLeader.py

```diff
@@ -131,8 +131,8 @@ def test(self):
 
         # 3 - Leader
         msg = leader_messages.next_mle_message(mle.CommandType.DATA_RESPONSE)
-        command.check_data_response(msg, network_data=CheckType.CONTAIN,
-            prefixes=[('2001:2:0:1::/64', 'paros'), ('2001:2:0:2::/64', 'paro')])
+        command.check_data_response(msg, network_data_opt=CheckType.CONTAIN,
+            prefixes=[ b'2001000200000001', b'2001000200000002' ])
```

Comment:
update to `network_data_check=...`- which should be reason for 7.1.3 failure in Travis.